src/pivotal-ui/components/meta/meta.scss

```diff
@@ -0,0 +1,141 @@
+  </symbol>
+
+  <symbol id="i-558-mailbox" viewBox="0 0 24 28">
+    <path d="M22 8H11V6h7l-2-2.913L18 0H9v8H2a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h10v4h4v-4h6a2 2 0 0 0 2-2V10a2 2 0 0 0-2-2zm0 14H4V10h5v4.277c-.596.347-1 .985-1 1.723a2 2 0 0 0 4 0c0-.738-.404-1.376-1-1.723V10h11v12z"/>
+  </symbol>
+</svg>
+
+<div class="row">
+  <div class="col-md-8">
+    <div class="panel panel-basic-alt bg-neutral-10 pvxxl phxl" id="need-help-expandable">
+
+      <div class="row">
+        <div class="col-xs-7 col-xs-offset-1 txt-c">
+          <img alt="support avatar" class="avatar" height="60" src="styleguide/avatar.svg" width="60">
+        </div>
+        <div class="col-xs-16">
+          <h4>Need help?</h4>
+          <p>We're here for you.</p>
+        </div>
+        <div class="col-xs-13 col-xs-offset-7 mtxl">
+          <a class="btn btn-default collapsed need-help type-neutral-4" data-toggle="collapse" href="#need-help-collapse-example" rel="noopener noreferrer" data-parent="need-help-expandable">
+            <div class="when-collapsed">
+              YES PLEASE!
+            </div>
+          </a>
+        </div>
+      </div>
+
+      <div class="collapse" id="need-help-collapse-example">
+        <ul class="list-unstyled">
+          <li class="row">
+            <div class="col-xs-7 col-xs-offset-1 txt-c ptm mbxl">
+              <!-- {{icon "@npmcorp/glyphish/g5/692-iphone-5.svg" class="icon icon-title"}} -->
+              <svg class="icon icon-title"><use xlink:href="#i-692-iphone-5"/></svg>
+            </div>
+            <div class="col-xs-16">
+              <h5 class="em-low mbn mtn">Call us!</h5>
+              <a class="type-neutral-1" href="tel:8443350303">844-335-0303</a>
+            </div>
+          </li>
+          <li class="row">
+            <div class="col-xs-7 col-xs-offset-1 txt-c ptm mbxl">
+              <!-- {{icon "@npmcorp/glyphish/g5/563-calendar.svg" class="icon icon-title"}} -->
+              <svg class="icon icon-title"><use xlink:href="#i-563-calendar"/></svg>
+            </div>
+            <div class="col-xs-16">
+              <h5 class="em-low mbn mtn">Let us call you</h5>
+              <a class="type-neutral-1" href="https://calendly.com/npmteam">Schedule a time</a>
+            </div>
+          </li>
+          <li class="row">
+            <div class="col-xs-7 col-xs-offset-1 txt-c ptm">
+              <!-- {{icon "@npmcorp/glyphish/g5/558-mailbox.svg" class="icon icon-title"}} -->
+              <svg class="icon icon-title"><use xlink:href="#i-558-mailbox"/></svg>
+            </div>
+            <div class="col-xs-16">
+              <h5 class="em-low mbn mtn">Email us!</h5>
+              <a class="type-neutral-1" href="mailto:sales@npmjs.com">sales@npmjs.com</a>
+            </div>
+          </li>
+        </ul>
+      </div>
+
+    </div>
+  </div>
+</div>
+
+```
+*/
+
+.need-help {
```

Comment:
do you need this? I think bootstrap has a specific class for this (and an animation if I'm not mistaken)